src/styles/pages/about.scss

```diff
@@ -1,38 +1,218 @@
 @import '../common';
 
 .PingCAP-About {
+  $is-12: 80vw;
+  $fixed-is-12: 84rem;
+
+  @mixin custom-column-section($columns: 12) {
```

Comment:
Is this function same with function `column-to-padding` ?

Reply:
no, they are not the same thing, but basically they do same work. I did not used custom-column-section in en about us, delete it later.

Reply:
> Is this function same with function `column-to-padding` ?

no, they are not the same thing, but basically they do same work. I did not used custom-column-section in en about us, delete it later.